src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -1148,6 +1155,9 @@ $top-buttons-spacing: 6px;
 	app-content will be shrinked properly
 */
 .app-sidebar {
+	--app-sidebar-width: clamp(300px, 27vw, 500px);
```

Comment:
Theoretically we support iOS 12.x (from our browserslist, from clients its 15.x only). So `clamp` does not work for iOS < 13.3

If we still support it, then:
```css
	--app-sidebar-width: 300px;
	--app-sidebar-width: clamp(300px, 27vw, 500px);
```

or
```css
	--app-sidebar-width: min(max(300px, 27vw), 500px);

Reply:
nevermind is 15.5 nowaday